packages/bpk-component-snippet/src/BpkSnippet.module.scss

```diff
@@ -0,0 +1,110 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@use '../../unstable__bpk-mixins/breakpoints';
+@use '../../unstable__bpk-mixins/tokens';
+@use '../../unstable__bpk-mixins/typography';
+
+.bpk-snippet {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+
+  @include breakpoints.bpk-breakpoint-small-tablet {
+    flex-direction: column;
+    gap: tokens.bpk-spacing-base();
```

Comment:
This will cause issues with Safari 14.0 which is our minimum supported browser version in Skyscanner - have you tested the output in this browser version?

Reply:
My Safari version is 17.5. I could not install the version 14. I use `user agent` development tool to test and it looks OK but i'm not sure this is the right way to test. could you please tell me how does other component test this?
![image](https://github.com/user-attachments/assets/5e91f2a4-5958-45e6-a779-be2dd9c3c51c)

Reply:
Here we can probably use SauceLabs to test the older browser version for the compatibility

Reply:
Thank you! i use the SauceLabs to test on Safari 14.0, and it looks good!
<img width="1495" alt="image" src="https://github.com/user-attachments/assets/41bd9c37-8c4d-4ee8-adf4-4fa3105bb42a">